Cql/Elm/PublicAPI.Unshipped.txt

```diff
@@ -813,8 +813,6 @@ static readonly Hl7.Cql.Elm.Library.JsonSerializerOptionsStrict -> System.Text.J
 ~Hl7.Cql.Elm.CodeFilterElement.value.set -> void
 ~Hl7.Cql.Elm.CodeFilterElement.valueSetProperty.get -> string
 ~Hl7.Cql.Elm.CodeFilterElement.valueSetProperty.set -> void
-~Hl7.Cql.Elm.CodeRef.libraryName.get -> string
```

Comment:
Was that an element we added in a partial class and that we have removed again?

Reply:
This was an accident. There is actually quite a lot of public members missing from this file from the generated ELM.g.cs. I'll put them all in

Reply:
This was an accident removal. I see a lot of methods are missing from the generated elm file, they have all been added.